lib/bump.rb

```diff
@@ -220,31 +250,6 @@ def find_version_file(pattern)
         end
       end
 
-      def next_version(current, part)
```

Comment:
What actually changed here? Indentation?

Reply:
@gregorym I've moved this method to public so we can call it from bin/bump